exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/UpsertStatement.kt

```diff
@@ -22,20 +25,25 @@ import org.jetbrains.exposed.sql.vendors.currentDialect
 open class UpsertStatement<Key : Any>(
     table: Table,
     vararg val keys: Column<*>,
-    val onUpdate: List<Pair<Column<*>, Expression<*>>>?,
+    val onUpdate: MutableList<Pair<Column<*>, Expression<*>>>?,
```

Comment:
It's minor, but I don't see any modifications of that list, so probably it should not be mutable

Reply:
Modification takes place in `upsert()`. To allow access to current `asForInsert()`, `UpsertStatement` is used as the receiver, so the new statement instance has to be passed to the constructor parameter somehow. I also would prefer to not do this, so I can maybe try  to create a new builder as the receiver.